### PR TITLE
Estute/configure ghprb plugin

### PR DESCRIPTION
**Pull request by estute**

### CODE REVIEW

File: e2e/test_plugin_installations.py [hunk 1]
```diff
@@ -19,38 +19,39 @@ def test_sample_plugins_installed(self):
...
             # the plugins installed via test_data/plugins.yml
             "node-iterator-api": "1.1",
             "github": "1.9.1",
             "plain-credentials": "1.1",
-            "credentials": "1.21",
-            "matrix-project": "1.6",
+            "credentials": "1.24",
+            "matrix-project": "1.4.1",
             "ssh-agent": "1.3",
-            "github-api": "1.82",
```
Comment (michaelyoungstrom) [inline e2e/test_plugin_installations.py]: I'm curious about why some of these plugins are the version they are. Almost all of them match build Jenkins (including GHPRB). But I'm seeing the following differences on Build Jenkins:
```
github-api 1.69
git 2.4.0
git-client 1.19.0
mapdb-api 1.0.6.0
```

**Comment by michaelyoungstrom** [comment]
LGTM once you get Travis passing

**Comment by estute** [comment]
@jzoldak @michaelyoungstrom I would like some input here. Does it seem like a good idea to have bokchoy extensively test every created data field on these configuration pages? Or are a few spot checks sufficient (mind you, we are also "testing" via codenarc and exception handling). This PR represents the latter option, but I can easily add in more tests.

**Comment by jzoldak** [comment]
@estute in answer to your question, I don't think that we need to or should extensively verify every option in the acceptance (bok-choy) test(s).

Also, I would recommend that we only have a single bok-choy acceptance test for each plugin. The way it is now is more unit-test style, where if something fails, it will be the only thing that fails and pinpoint exactly what the root cause is. The downside is that the test suite will take longer to execute, as it will be opening a new browser instance and navigating to the configuration page multiple times (as many times as there are tests). 

Instead I would prefer a single test that verifies a few things and will fail at the first failed assertion. I think that level of granularity will be plenty for us to figure out what the problem is.

**Comment by michaelyoungstrom** [comment]
LGTM

**Pull request merged**